packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.html

```diff
@@ -42,6 +43,14 @@
                             aria-hidden="true"
                         ></span>
                     </label>
+                    <div style="height:24px;">
```

Comment:
**question:** What is the need to set the custom height?

Reply:
I don't know about this but anyway i removed it to use a margin